src/main/java/in/org/projecteka/hdaf/link/HIPClient.java

```diff
@@ -1,14 +1,16 @@
 package in.org.projecteka.hdaf.link;
 
-import in.org.projecteka.hdaf.link.link.model.PatientLinkReferenceRequest;
+import in.org.projecteka.hdaf.link.link.ClientError;
```

Comment:
This is kind of deviating from what do we need right, using `ClientError` from the other package?